lib/modules/uscore_v3.1.1/bodyheight_sequence.rb

```diff
@@ -259,6 +265,10 @@ def patient_ids
           reply = perform_search_with_status(reply, search_params) if reply.code == 400
 
           validate_search_reply(versioned_resource_class('Observation'), reply, search_params)
+          resource_returned = fetch_all_bundled_resources(reply, check_for_data_absent_reasons)
```

Comment:
Is there any way to do this without paging through all the results again?  Our tests are already pretty slow, and this adds more requests.  Can we just move this assert into `validate_search_results`?